persist/fs/seek.go

```diff
@@ -221,14 +213,39 @@ func (s *seeker) Open(namespace ts.ID, shard uint32, blockStart time.Time) error
 		return err
 	}
 
+	s.indexLookup, err = newNearestIndexOffsetLookupFromSummariesFile(
+		summariesFd,
+		s.summariesFdWithDigest,
+		s.expectedSummariesDigest,
+		s.decoder,
+		int(s.summariesInfo.Summaries),
+	)
+	if err != nil {
+		s.Close()
+		return err
+	}
+
 	if !s.keepUnreadBuf {
 		// NB(r): Free the unread buffer and reset the decoder as unless
 		// using this seeker in the seeker manager we never use this buffer again
 		s.unreadBuf = nil
 		s.decoder.Reset(nil)
 	}
 
+	// Make sure the indexFd is at the beginning of the file
+	_, err = indexFd.Seek(0, 0)
```

Comment:
Do we need to seek to zero here?  It doesn't look like anything uses the `indexFd` before the mmap call?

Reply:
you're right, I'm not sure why I put this there. Maybe just being overly precautious